benchmark/cases/better_case.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/ev python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Example performance test using benchmark framework. Test react speed on amount of incoming messages using normal agent operating."""
+import time
+
+from benchmark.cases.helpers.dummy_handler import DummyHandler
+from benchmark.framework.aea_test_wrapper import AEATestWrapper
+from benchmark.framework.benchmark import BenchmarkControl
+from benchmark.framework.cli import TestCli
+
+from aea.configurations.base import SkillConfig
+
+
+def make_agency_conf(name: str = "dummy_agent", skills_num: int = 1) -> dict:
```

Comment:
Can we call this `make_custom_config` or something like that. I don't think the `agency` bit is helpful and also we want to clarify that this config is specific to the test case wrapper.

Reply:
Note, the docstring is incomplete, but probably best to make this a private method and keep the docstring as is.

Reply:
renamed. and it's just a helper function, it's not assumed to be reused outside of the test.